opencsp/common/lib/render_control/RenderControlTower.py

```diff
@@ -0,0 +1,47 @@
+import opencsp.common.lib.render_control.RenderControlPointSeq as rcps
+import opencsp.common.lib.render_control.RenderControlText as rctxt
+from opencsp.common.lib.render_control.RenderControlPointSeq import RenderControlPointSeq
+
+
+class RenderControlTower:
+    """
+    Render control for towers.
+    """
+
+    def __init__(
+        self,
+        centroid: bool = False,
+        draw_name=False,
+        name_style=rctxt.default(color='k'),
+        draw_outline=True,
+        point_styles: RenderControlPointSeq = rcps.marker(),
```

Comment:
It's considered bad form to use mutable default arguments in function definitions. The reason is because the mutable value might change, and then the function won't behave as expected. See here for a more in-depth description of the problem: https://www.codecademy.com/learn/learn-intermediate-python-3/modules/int-python-function-arguments/cheatsheet

The better option is to use a default value of None and then assign the mutable value inside the body of the function. For example:

```python
def __init__(
    self,
    point_styles: RenderControlPointSeq = None
) -> None:
    # default values
    if point_styles is None:
        point_styles = rcps.marker()
```

Reply:
That makes sense, thank you!